### PR TITLE
LOG-1863: Token for API call

### PR DESCRIPTION
**Pull request by apeksha-khandelwal**
[Log 1863](https://issues.redhat.com/browse/LOG-1863)

Added an ability to provide a token for every API request.

### CODE REVIEW

File: pkg/cmd/historical_logs.go [hunk 1]
```diff
@@ -148,14 +149,23 @@ func FetchLogs(baseUrl string, logParameters *LogParameters, podname string, pod
...
+
 	query := req.URL.Query()
-	query.Add("podname", podname)
-	query.Add("namespace", logParameters.Namespace)
-	query.Add("starttime", logParameters.StartTime)
-	query.Add("finishtime", logParameters.EndTime)
-	query.Add("maxlogs", strconv.Itoa(logParameters.Limit))
-	query.Add("level", logParameters.Level)
+	query.Add("/pod/:", podname)
+	query.Add("/namespace/:", logParameters.Namespace)
```
Comment (igor-karpukhin) [inline pkg/cmd/historical_logs.go]: @apeksha-khandelwal please remove these `:` in all calls. Also, I'm not sure if this one builds the URL correctly. Will the URL be in the correct order if one specifies only the pod name?

File: pkg/cmd/historical_logs.go [hunk 2]
```diff
@@ -148,14 +149,23 @@ func FetchLogs(baseUrl string, logParameters *LogParameters, podname string, pod
...
+	query.Add("/starttime/:", logParameters.StartTime)
+	query.Add("/finishtime/:", logParameters.EndTime)
+	query.Add("/maxlogs/:", strconv.Itoa(logParameters.Limit))
+	query.Add("/level/:", logParameters.Level)
 	req.URL.RawQuery = query.Encode()
+	fmt.Println(req.URL)
```
Comment (igor-karpukhin) [inline pkg/cmd/historical_logs.go]: Please remove this debug statement

**Review by igor-karpukhin — CHANGES_REQUESTED** [review]
@apeksha-khandelwal Looks good in general. Please rebase this PR and then we can merge it.

File: pkg/cmd/historical_logs.go (hunk 2 above)
Comment (apeksha-khandelwal) [reply pkg/cmd/historical_logs.go]: I updated the code to add parameters and removed the debugging statement.

**Review by igor-karpukhin — APPROVED** [review]
Looks good

**Pull request merged**